ompy/extractor.py

```diff
@@ -153,6 +174,21 @@ def extract_from(self, ensemble: Optional[Ensemble] = None,
         self.nld = nlds
         self.gsf = gsfs
 
+        if self.error_estimator is not None and regenerate:
+            LOG.debug("Estimating relative errors")
+            nld_rel_err, gsf_rel_err = self.error_estimator(nlds, gsfs)
+            nld_rel_err.to_keV()
+            gsf_rel_err.to_keV()
+            for i, (nld, gsf) in enumerate(zip(self.nld, self.gsf)):
+                nld.std = self.rel_err_missing * nld.values
+                gsf.std = self.rel_err_missing * gsf.values
+                idx_nld = nld.indices(nld_rel_err.E)
+                idx_gsf = gsf.indices(gsf_rel_err.E)
+                nld.std[idx_nld] = nld_rel_err.values * nld.values[idx_nld]
+                gsf.std[idx_gsf] = gsf_rel_err.values * gsf.values[idx_gsf]
+                nld.save(self.path / f'nld_{i}.npy')  # Overwrite with errors!
+                gsf.save(self.path / f'gsf_{i}.npy')  # Overwrite with errors!
```

Comment:
what is the meaning of the inline comment? is this supposed to happen here (but not yet...)?

Reply:
Regardless of attribute `error_estimator` is set or not, the extracted NLD/GSF will be written to disk here: https://github.com/vetlewi/ompy/blob/1ab2618ec0c2b2e7b8c5935242183587e3451f00/ompy/extractor.py#L169-L170

If `error_estimator` is set then these two lines will overwrite the generated files on the disk. Could change the comment to `Update files on disk with errors` or something similar